mathesar/api/ui/serializers/users.py

```diff
@@ -80,14 +80,21 @@ def validate_old_password(self, value):
             return value
         raise IncorrectOldPassword(field='old_password')
 
+    def validate_password(self, value):
+        try:
+            validate_password(value)
+        except serializers.ValidationError as exc:
```

Comment:
I suspect that this won't work because you are trying to catch the `ValidationError` that is defined in DRF's exceptions module while `validate_password()` raises the one defined in `django.core.exception` in case of a failure. Can you please fix this?